sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/schema.yaml

```diff
@@ -1046,3 +1046,9 @@ fields:
   mode: REPEATED
   name: scalar_parent_urlbar_searchmode_typed_sum
   type: RECORD
+- mode: NULLABLE
+  name: scalar_parent_os_environment_launch_method
+  type: STRING
```

Comment:
I now see https://github.com/mozilla/bigquery-etl/pull/1953#discussion_r611970175 which explains the use case, but was hidden as outdated.

I think this would be easier to work with and more efficiently stored as a set of five distinct boolean fields, and would still enable the use case Su was talking about.